app/graph/types/project_media_type.rb

```diff
@@ -85,9 +85,14 @@
 
   field :annotations_count do
     type types.Int
+    argument :annotation_type, types.String
 
-    resolve ->(project_media, _args, _ctx) {
-      project_media.get_versions_log_count
+    resolve ->(project_media, args, _ctx) {
```

Comment:
do we have a guarantee that `project_media.get_versions_log_count === project_media.get_annotations([all types]).count` ?